packages/go-bin/go-bin.pacscript

```diff
@@ -0,0 +1,74 @@
+maintainer="Paul Cosma (saenai) <paul.cosma97@gmail.com>"
+
+name="go-bin"
+pkgname="go"
+version="1.17.6"
+description="Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
+url="https://go.dev/dl/go${version}.linux-amd64.tar.gz"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-app ${pkgname}-deb golang golang-git golang-deb golang-bin golang-app golang-go"
+gives="golang-go"
+replace="golang-go"
+hash="231654bbf2dab3d86c1619ce799e77b03d96f9b50770297c8f4dff8836fc8ca2"
+
+__goroot="/usr/local/${name}"
+__shell_rcs=("/home/$(whoami)/.bashrc" "/home/$(whoami)/.zshrc" "/home/$(whoami)/.profile")
+
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  # Install binary
+  sudo install -Dm755 "bin/go" "${STOWDIR}/${name}/usr/bin/go"
+  sudo install -Dm755 "bin/gofmt" "${STOWDIR}/${name}/usr/bin/gofmt"
+
+  sudo mkdir -p "${__goroot}"
+  sudo cp -r "." "${__goroot}"
+  sudo chmod -R 755 "${__goroot}"
```

Comment:
Why moving things to outside `STOWDIR`?

Reply:
Just to be sure I got it right, `STOWDIR` is used for binary symlinks, right? I needed so save Go's libs in a persistent dir

Reply:
Everything you put in stowdir get's symlinked to the system. It could include libraies, desktop files, binaries. Anything except debs.

Reply:
@saenai255 what about this?

Reply:
@saenai255